tests/microgrid/test_graph.py

```diff
@@ -816,8 +823,11 @@ async def test_refresh_from_api(self) -> None:
         servicer.set_connections([(101, 111), (111, 131)])
         await graph.refresh_from_api(client)
 
+        grid_max_current = Current.from_amperes(0.0)
```

Comment:
We can use the quantity zero constructor instead as follows:
```suggestion
        grid_max_current = Current.zero()
```

Reply:
Nice! Done